Model/Page.php

```diff
@@ -134,6 +134,11 @@
     protected $target;
 
     /**
+     * @var int
+     */
+    protected $targetRedirectType = 302;
```

Comment:
I never use protected fields with doctrine. If there is a getter and a setter, it should work, shouldn't it?